client/src/controllers/index.ts

```diff
@@ -59,4 +60,5 @@ export const coreControllerDefinitions: Definition[] = [
   { controllerConstructor: TooltipController, identifier: 'w-tooltip' },
   { controllerConstructor: UnsavedController, identifier: 'w-unsaved' },
   { controllerConstructor: UpgradeController, identifier: 'w-upgrade' },
+  { controllerConstructor: BlockController, identifier: 'w-block' },
```

Comment:
Please add the controller registration lines alphabetically, as per the comments at the start of this section.